policies_rosa/assembly-policy-process-security.adoc

```diff
@@ -1,8 +1,7 @@
-include::_attributes/attributes-openshift-dedicated.adoc[]
```

Comment:
If this has been published in the old location, you need to work with Vikram about the redirect.

Reply:
This will be ok as it's in the new repo now.